test/minikube/minikube_long_restore_test.go

```diff
@@ -791,3 +793,93 @@ func TestKubernetesRestoreDatabaseSeparateJournal(t *testing.T) {
+	defer testlib.VerifyTeardown(t)
+
+	options := helm.Options{
+		SetValues: map[string]string{
+			"database.name":                         "demo",
+			"database.archiveType":                  archiveType,
+			"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"backup.persistence.enabled":            "true",
+			"backup.persistence.size":               "1Gi",
+			"database.te.logPersistence.enabled":    "true",
+		},
+	}
+
+	randomSuffix := strings.ToLower(random.UniqueId())
+	namespaceName := fmt.Sprintf("%skubernetesrestoredatabase-%s", testlib.NAMESPACE_NAME_PREFIX, randomSuffix)
+	testlib.CreateNamespace(t, namespaceName)
+
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	helmChartReleaseName, _ := testlib.StartAdmin(t, &options, 1, namespaceName)
+
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+	databaseChartName := testlib.StartDatabase(t, namespaceName, admin0, &options)
+
+	opts := k8s.NewKubectlOptions("", "", namespaceName)
+	options.KubectlOptions = opts
+
+	opt := testlib.GetExtractedOptions(&options)
+
+	// Generate diagnose in case this test fails
+	testlib.AddDiagnosticTeardown(testlib.TEARDOWN_DATABASE, t, func() {
+		testlib.GetDiagnoseOnTestFailure(t, namespaceName, admin0)
+		testlib.RecoverCoresFromEngine(t, namespaceName, "te",
+			fmt.Sprintf("%s-nuodb-%s-%s-log-te-volume", databaseChartName, opt.ClusterName, opt.DbName))
+	})
+
+	tePodNameTemplate := fmt.Sprintf("te-%s-nuodb-%s-%s", databaseChartName, opt.ClusterName, opt.DbName)
+	smPodNameTemplate := fmt.Sprintf("sm-%s-nuodb-%s-%s", databaseChartName, opt.ClusterName, opt.DbName)
+	tePodName := testlib.GetPodName(t, namespaceName, tePodNameTemplate)
+	smPodName0 := testlib.GetPodName(t, namespaceName, smPodNameTemplate)
+
+	// Execute initial backup
+	backupGroup0 := fmt.Sprintf("%s-0", opt.ClusterName)
+	testlib.BackupDatabase(t, namespaceName, smPodName0, opt.DbName, "full", backupGroup0)
+
+	t.Run("restoreDatabaseSameVersion", func(t *testing.T) {
+		testlib.CreateQuickstartSchema(t, namespaceName, admin0)
+		go testlib.GetAppLog(t, namespaceName, tePodName, "_same_pre-restart", &corev1.PodLogOptions{Follow: true})
+		go testlib.GetAppLog(t, namespaceName, smPodName0, "_same_pre-restart", &corev1.PodLogOptions{Follow: true})
+
+		// restore database
+		defer testlib.Teardown(testlib.TEARDOWN_RESTORE)
+		testlib.RestoreDatabase(t, namespaceName, admin0, &options)
+
+		go testlib.GetAppLog(t, namespaceName, smPodName0, "_same_post-restart", &corev1.PodLogOptions{Follow: true})
+
+		if archiveType == "lsa" {
+			require.False(t, testlib.HasFile(t, namespaceName, smPodName0, "/var/opt/nuodb/archive/nuodb/demo/10.atm"))
```

Comment:
If `1.atm` file doesn't exist in LSA archives, then we need to also update the _autoImport_ gate in `nuosm`. See https://github.com/nuodb/nuodb-helm-charts/blob/master/stable/database/files/nuosm#L497.

Reply:
done.